tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -746,3 +748,134 @@ def test_02_multiattach_volumes(self):
                     reporting.add_test_script(test[0])
                     reporting.test_case_to_write()
 
+    # Workload with scheduler and retention parameter
+    @decorators.attr(type='workloadmgr_cli')
+    def test_03_multiattach(self):
+        reporting.add_test_script(str(__name__) + "_retention_with_multiattach_volume")
+        try:
+            retention = 3
+            start_date = time.strftime("%m/%d/%Y")
+            start_time = (datetime.now() + timedelta(minutes=5)
+                          ).strftime("%I:%M %p")
+            self.schedule = {
+                "fullbackup_interval": "0",
+                "retention_policy_type": "Number of Snapshots to Keep",
```

Comment:
use tvaultconf.retention_policy_type instead of hardcoding

Reply:
done